enterprise/app/trends/trends.tsx

```diff
@@ -354,6 +354,18 @@ export default class TrendsComponent extends React.Component<Props, State> {
                 separateAxis={true}
               />
 
+              <TrendsChartComponent
```

Comment:
please condition this on `capabilities.config.trendsSummaryEnabled` so that we push both of these when the data is actually available (so, wrap this in `{capabilities.config.trendsSummaryEnabled && (<TrendsChartComponent...>)`